tests/director/api/fixtures_queries_test.go

```diff
@@ -10,6 +10,21 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func SetAutomaticScenarioAssignmentWithinTenant(t *testing.T, ctx context.Context, tenantID, automaticScenario, selecterKey, selectorValue string) {
```

Comment:
Apart from the typo we already have 2(???) fixtures with  similar functionality, please decide which is the best one and clean up the rest.

Reply:
[Here](https://github.com/kyma-incubator/compass/blob/86d9878d6bb98a35e05a1b21aeb6a081a7999475/tests/director/api/fixtures_queries_test.go#L461) and [here](https://github.com/kyma-incubator/compass/blob/86d9878d6bb98a35e05a1b21aeb6a081a7999475/tests/director/api/fixtures_queries_test.go#L472)